src/components/Users.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const User = props => (
```

Comment:
Делай деструктуризацию props
```suggestion
const User = ({ postAuthor }) => (
```